source/server/server.cc

```diff
@@ -15,6 +15,7 @@
 #include "envoy/upstream/cluster_manager.h"
 
 #include "common/api/api_impl.h"
+#include "common/common/backward.h"
```

Comment:
We don't currently, we just use the built-in libevent signal processing functionality for SIGTERM, SIGHUP, etc. We let Envoy crash and dump and then have an external process which sucks up the core dump and does stuff with it.

I think it makes sense to document which signals Envoy itself handles and then just allow people to override things like SIGABRT, SIGSEGV, etc. in main if they want. I'm open to whatever solution here makes the most sense. We could have a stub signal setup library that is called from main that is empty in the OSS build, and you could override?

Also, while we are on this topic, my vote would be to make the backtrace stuff opt-in on production builds, and default on debug builds.